### ISSUE TITLE
Can we forego the need to close and re-open pull request upon signing the CLA?

### ISSUE THREAD

**Issue by filmaj** [adobe/cla-bot#55]
I think we'd need the Adobe Sign API to have some kind of event or webhook to trigger a check on our side, right?

**Comment by hirenoble** [comment]
Yes, Adobe Sign APIs do have concept of webhook but I recall there was some challenge when we discussed about this earlier and then decided to go for the close and reopen the pull request. But we can discuss more on what's possible.

**Comment by filmaj** [comment]
The CLA Assistant experience when sending a pull request to a github.com/magento repo is nice because it has none of the close/reopen - a nice streamlined experience.

**Comment by hirenoble** [comment]
Nice! We can take some inspiration! :)

**Comment by hirenoble** [comment]
Ok got it. So, Magento uses CLA Assistant for the CLA check. Which means we might have to go away from Adobe Sign and use CLA Assistant instead.

**Comment by filmaj** [comment]
We could configure a Sign webhook to trigger when Agreements get created and then deliver the Agreement Info to a new serverless action.

This action could then:
- use the same code in `lookup` to parse form data for the data agreement to extract github username(s)
- use the GitHub Search API to look for open pull requests in the adobe/magento/all installed orgs that were authored by said github usernames
- post updates on the pull requests identified.

**Comment by filmaj** [comment]
@hirenoble using CLA Assistant won't work as we need the specific CLA document signed as per legal requirements. CLA Assistant isn't much more than an authorization via GitHub Oauth - no actual document is signed.

**Comment by filmaj** [comment]
The above issue has been resolved (at least, a workaround was found) and I am unblocked here again. I continue onwards.

**Comment by filmaj** [comment]
Well, hit another issue, this time it looks like im getting 'agreement not found' when I try to leverage the Adobe Sign search API using the agreement ID the webhook is receiving from signing event payloads... off to ask the Sign team again.

**Comment by filmaj** [comment]
Got unblocked, need  to use the v6 version of the search API with IDs returned by the webhook event payload (which is also v6). Don't cross API versions!

**Comment by filmaj** [comment]
Hit an issue where apparently passing parameters between actions is not working as expected... on top of that it seems like the Adobe Sign webhook payload data structure changed today... sigh... I continue to debug and roll with the punches.

**Comment by filmaj** [comment]
I redeployed the action and now things seem to be working....
